src/util/helperUtil.ts

```diff
@@ -39,12 +42,33 @@ export function getConfiguration<T>(section: string, resourceOrFilepath?: vscode
  * @returns a list of projects that the given command can be excuted on.
  */
 export function filterProjects(projects: LibertyProject[], command: string): LibertyProject[] {
-	let resultProjects: LibertyProject[] = [];
+	const resultProjects: LibertyProject[] = [];
 	for ( const project of projects) {
 		const applicableTypes = COMMAND_AND_PROJECT_TYPE_MAP[command];
 		if (applicableTypes.includes(project.getContextValue())) {
 			resultProjects.push(project);
 		}
 	}
 	return resultProjects;
-}
\ No newline at end of file
+}
+
+
+/**
+ * Gets the stored dashboard data.
+ * @param context 
+ * @returns 
+ */
+export function getStorageData(context: vscode.ExtensionContext): DashboardData {
+	let data = context.workspaceState.get<DashboardData>(LIBERTY_DASHBOARD_WORKSPACE_STORAGE_KEY, new DashboardData([]));
+	data = new DashboardData(data.projects);
+	return data;
+}
+/**
+ * Stores the dashboard data to workspace storage
+ * @param context
+ * @param dasboardData 
+ */
+export async function saveStorageData(context: vscode.ExtensionContext, dasboardData: DashboardData): Promise<void>{
+	await context.workspaceState.update(LIBERTY_DASHBOARD_WORKSPACE_STORAGE_KEY, dasboardData);
```

Comment:
The following flow does not seem to persist the Liberty dashboard projects:
1. Open multiple projects in VS Code
2. Manually add Liberty projects to the dashboard
3. Save as a workspace configuration, close the window
4. Open the window, the manually add projects are not persisted.

This is not in debugging mode. I observe this when building the extension with `vsce package` and installing to my VS Code.

Reply:
It works for me.

https://user-images.githubusercontent.com/35569780/194130423-b272b19a-e30a-4c76-9406-18a4959dd314.mov